internal/exec/stages/files/files.go

```diff
@@ -83,16 +93,18 @@ func (s stage) Run(config types.Config) error {
 		return fmt.Errorf("failed to create units: %v", err)
 	}
 
-	if err := s.createCrypttabEntries(config); err != nil {
-		return fmt.Errorf("creating crypttab entries: %v", err)
-	}
+	if !s.isLive {
+		if err := s.createCrypttabEntries(config); err != nil {
+			return fmt.Errorf("creating crypttab entries: %v", err)
+		}
 
-	if err := s.createResultFile(); err != nil {
-		return fmt.Errorf("creating result file: %v", err)
-	}
+		if err := s.createResultFile(); err != nil {
```

Comment:
It's not clear to me that we should skip this.  It's still useful to record that Ignition has applied customizations to the image.

Reply:
Hmm, I think that might end up being confusing. First, it's fine to run this tool multiple times, so we can end with multiple nested configs. Second, baking it in means it'll be part of the OSTree commit which can cause issues since it'll conflict with the real boot case. We don't currently build bootimages from these containers but could in the future. At the very least, currently this would make the report in the actual booted system show up as a modified file in `ostree admin config-diff`. Also, the report contains some things that don't really apply well to the container path (e.g. `provisioningBootID` isn't really useful, and `userConfigProvided` is always `true`).

That said, at a high level with CoreOS layering (at least within OCP) we're hoping you'll be able to easily visualize applications of Ignition, e.g. by capturing its run in a separate layer. But... we might circle back to this because indeed having Ignition itself write a report could play a role in how we present applied changes to the user, but it'll require some tweaks (e.g. put in a separate path, or folded into the OSTree commit metadata, and possibly different fields).

Reply:
Okay, fair.  I'm okay with omitting for now, and revisiting the report design later on.